### ISSUE TITLE
Develop a plan for reaching out to young professionals

### ISSUE THREAD

**Issue by dasaderi** [OpenCon-Cascadia/organization#5]
Pam, Ashley, Daniela, Emily and Sam
**The question is what is the most effective way that we can reach out to early career science researchers in this region, let them know the OC is happening in January, and get information back from them about what kind of mentoring they would like at the Con?**

Emily F.: Solicit the kind of mentorship needed is via the registration form. A question on the form could include a question about what kind of peer support or non peer support folks feel like they might need. 

Ashley F.: Reach out to our communities, e.g. OpenCon Librarian's call, Twitter, ECR eLIFE, etc. Also check out https://www.alimb.ca/

Daniela S.: Offer peer review mentorship. PREreview is developing a mentorship program based on Mozilla Open Leaders. We can test this as a workshop at the conference?

**Comment by dasaderi** [comment]
Should we schedule a call to discuss this strategy?

**Comment by PJP1313** [comment]
The week of July 16th would be best for a call for me. What times and days work for everyone?

**Comment by ashleyfarley** [comment]
I agree that the week of July 16th works best. I am very flexible on days/times and can work around other's schedule.

**Comment by dasaderi** [comment]
Great! I'll email the group shortly to schedule a call. Thank you!

**Comment by doramaar** [comment]
I can't make the call but am still interested in helping!

**Comment by dasaderi** [comment]
July 12 – Call to plan how to reach out to young professionals and metorship roundtable's topics:
Who's on the call? Daniela, Ashley, Pam

**How should we reach out?**
- Tap on our networks and try to avoid only reaching out to immediate one.
- In the registration form for the conference ask for what kind of mentorship would they want to receive.
- In the registration form also ask for what method they prefer to continue the conversation after the meeting (*e.g*., mailing list, Slack channel).

**What content should we offer as possible topic for roundtable/mentorship workshop?**
- Tips on how open science can advance your career
- How to navigate the open science global space, where to find opportunities, and how to create a supporting network
- Code of conducts/lab manifestos for academic labs: making sure expectations are clear on both sides (Robin Champieux and Alison Hill, and [myself and another student](https://github.com/dasaderi/CoC_templates) have separately started efforts in this space and need to join forces).
- How to highlight non-research related activities and/or open science efforts in your CV/resume/elevator pitch.

**Comment by doramaar** [comment]
Regarding the last bullet point on content to offer... is this related to digital identity? Would there be interest by participants in learning about identity management in relation to open? (I'm thinking here about things like academia.edu, researchgate, ORCID, ImpactStory, etc....)

**Comment by dasaderi** [comment]
@doramaar that's a great idea! I'm totally in for that kind of discussion. In the new PREreview 2.0 platform we will be experimenting a little with that issue, and I'd love to get people's perspective on digital identities, particularly how to navigate open/anonymous identities when signing reviews in the context of wanting to push for shift in reward systems and incentives.

**Comment by dasaderi** [comment]
Hello you all. We are putting together the registration form (see working gdoc with agenda notes). We need to decide what to ask ECRs about interest with mentorship events/workshop we want to propose for the conference.

**Comment by pamelapierce** [comment]
Definitely at least one multiple choice question that asks them what form of mentorship they would prefer: 1. Panel 2. One on one conversations. 3. Reception.  Those are just a few possible options.

**Comment by doramaar** [comment]
I agree with above, but I wonder if we could use some different language aside from panel. To me, panels are impersonal, and mostly like a "sage on the stage" of people sharing their experience with limited time for questions. I feel like a word to signal something more interactive would be better... Workshop? Roundtable? Discussion?

**Comment by pamelapierce** [comment]
Do we want to have a short call at some point to discuss our reaching out to young professionals plan in more details?

**Comment by dasaderi** [comment]
Hi. Sorry, I was gone for a conference. I can get on a call next week to finalized questions for registration form. I'll include a link to a doodle in an email I'm prepping for the group. :)

**Comment by dasaderi** [comment]
So I was thinking that since we asked at registration if participants want to be mentored or want to provide mentorship, perhaps we can print that info on name tags or prompt participants to write their position on the name tag and add a sticker about this mentorship thing.

**Comment by rchampieux** [comment]
And/Or this could be one thing we try to facilitate at lunch for those that are interested.  Additionally, there aren't that many people that want to be mentored - it might be nice if we sent a special welcome to these attendees and asked them what sort of mentorship their looking for and what they are most excited about the conference.
